wiki/Twitter/ro.md

```diff
@@ -0,0 +1,24 @@
+# Twitter
+
+osu! are mai multe conturi de Twitter, fiecare având scopuri particulare. Majoritatea au un flux redus de postări, dar vă vor ajuta să vă țineți la curent în afara ecosistemului osu!
+
+## Servicii
+
+| Avatar | Handle | Descriere |
+| :-: | :-: | :-- |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Notificări cu trafic redus privind problemele legate de site-ul web și [Bancho](/wiki/Bancho_(server)). |
+| ![osu! support avatar](img/osusupport.jpg) | [@osusupport](https://twitter.com/osusupport) | Ajutor pentru rezolvarea problemelor legate de cont și/sau de comunitate. Administrat de [echipa de suport](/wiki/People/The_Team/Account_support_team). |
+
+## Comunitate
+
+| Avatar | Handle | Descriere |
+| :-: | :-: | :-- |
+| ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | Sursa oficială de știri și anunțuri. |
+| ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Postări comice când lucrurile eșuează. |
+| ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | Știri, anunțuri și scurte sondaje de opinie ale comunității de către [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (nu este condus de [echipa osu!](/wiki/People/The_Team)). |
```

Comment:
the table is missing the pp committee twitter row: https://github.com/ppy/osu-wiki/blob/master/wiki/Twitter/en.md?plain=1#L19

Reply:
ooh, missed that one. will fix now.